packages/web-frontend/src/utils/periodGranularities.js

```diff
@@ -27,34 +27,39 @@ const CONFIG = {
     chartFormat: 'Do MMMM YYYY',
     rangeFormat: 'Do MMMM YYYY',
     pickerFormat: 'D',
+    urlFormat: 'Do_MMM_YYYY',
```

Comment:
These formats should probably get chosen by someone else. I've just put place holders in.

Reply:
Fair enough! Let's add some information in the card about how the urls will look like, and then we can all decide